oauthproxy.go

```diff
@@ -667,6 +678,24 @@ func (p *OAuthProxy) SignOut(rw http.ResponseWriter, req *http.Request) {
 func (p *OAuthProxy) OAuthStart(rw http.ResponseWriter, req *http.Request) {
 	prepareNoCache(rw)
 
+	params := mux.Vars(req)
+	idString := (params["id"])
+
+	var providerSlice int
+	var ok bool
+
+	if idString == "" {
+		logger.Printf("Path Parameter empty: setting provider as default provider 0:")
```

Comment:
I don't think we need this log line, if you are keen to keep it, then I'd like to see the actual providerID within it rather than the `0`